String.php

```diff
@@ -462,6 +462,101 @@ public function toUpperCase ( ) {
     }
 
     /**
+     * Transform a UTF-8 string into an ASCII one.
+     * First, try with a transliterator. If not available, will fallback to a
+     * normalizer. If not available, will try something homemade.
+     *
+     * @access  public
+     * @param   bool  $try    Try something if \Normalizer is not present.
+     * @return  \Hoa\String
+     * @throw   \Hoa\String\Exception
+     */
+    public function toAscii ( $try = false ) {
+
+        if(0 === preg_match('#[\x80-\xff]#', $this->_string))
+            return $this;
+
+        $string = $this->_string;
+
+        $transId = 'Any-Latin; ' .
+                   '[\p{S}] Name; ' .
+                   'Latin-ASCII';
+
+        if(null !== $transliterator = static::getTransliterator($transId)) {
```

Comment:
It's not just adding an "elseif (false === $try) {throw new Exception(...);}" case to this `if`?
I don't know if you want to do that, but that was my point.